contributing/schema-changes.rst

```diff
@@ -140,5 +144,5 @@ diff` reports should help you to create a new
 :file:`sql/psql/OMERO5.1DEV__5/OMERO5.1DEV__4.sql`.
 
 When you commit your code and issue a pull request, include the
-changes to :omero_source:`omero.properties <etc/omero.properties>` and
+changes to :omero_model_source:`omero-model.properties <src/main/resources/omero-model.properties>` and
```

Comment:
Outwith this PR, in the longer term I wonder if `build-schema` should just be part of the normal build and release process so it goes in the zip but doesn't have the autogen output in version control.

Reply:
> but doesn't have the autogen output in version control.

No strong objection. There are times where I've `git grep`'d or `git log`'d on that directory, but that can be worked around.

Reply:
build-schema needs to be moved "down" in the stack. As discussed we do not need it for 5.5, but one thing  to look after.